src/validops.cu

```diff
@@ -76,6 +100,9 @@ size_t count_valid_bits_host(std::vector<gdf_valid_type> const & masks, const in
 gdf_error gdf_count_nonzero_mask(gdf_valid_type const * masks, int num_rows, int * count)
 {
 
+  // Why am I getting an unused function warning error if I don't do this?
+  gdf_is_valid(nullptr, 0);
```

Comment:
Because `gdf_is_valid` is declared `static` but not also `inline`. In PR #152 I've changed it to just `inline` -- I see no reason for it to be static. This warning/error only happened for me on g++7.x.